source/includes/_budget.md

```diff
@@ -2,31 +2,43 @@
+category_id         | number | true | Unique identifier for category
+category_group_name | string | true | Name of the category group, if applicable
+group_id            | number | true | Unique identifier for category group
+is_group            | boolean | true | If true, this category is a group
+is_income           | boolean | false | If true, this category is an income category (category properties are set in the app via the Categories page)
+exclude_from_budget | boolean | false | If true, this category is excluded from budget (category properties are set in the app via the Categories page)
+exclude_from_totals | boolean | false | If true, this category is excluded from totals (category properties are set in the app via the Categories page)
+data                | Array of data objects | false | For each month with budget or category spending data, there is a data object with the key set to the month in format YYYY-MM-DD. For properties, see Data object below.
+config              | object | true | Object representing the category's budget suggestion configuration
 
 ## Data Object
 
-Attribute Name      | Type   | Description
-------------------- | ----   | -----------
-budget_amount       | number | The budget amount, as set by the user. If empty, no budget has been set.
-budget_currency     | string | The budget currency, as set by the user. If empty, no budget has been set.
-budget_to_base      | number | The budget converted to the user's primary currency. If the multicurrency feature is not being used, budget_to_base and budget_amount will be the same. If empty, no budget has been set.
-spending_to_base    | number | The total amount spent in this category for this time period in the user's primary currency
-num_transactions    | number | Number of transactions that make up "spending_to_base"
-is_automated        | boolean | If true, the budget_amount is only a suggestion based on the set config. If not present, it is false (meaning this is a locked in budget)
+Attribute Name      | Type   | Nullable | Description
+------------------- | ----   | -------- | -----------
+budget_amount       | number | true | The budget amount, as set by the user. If empty, no budget has been set.
+budget_currency     | string | true | The budget currency, as set by the user. If empty, no budget has been set.
+budget_to_base      | number | true | The budget converted to the user's primary currency. If the multicurrency feature is not being used, budget_to_base and budget_amount will be the same. If empty, no budget has been set.
+spending_to_base    | number | false | The total amount spent in this category for this time period in the user's primary currency
+num_transactions    | number | false | Number of transactions that make up "spending_to_base"
+is_automated        | boolean | true | If true, the budget_amount is only a suggestion based on the set config. If not present, it is false (meaning this is a locked in budget)
```

Comment:
i suppose i should return true or false, instead of true or null as this is suggesting!

Reply:
let's leave this as is for now!